spec/unit/block_score/person_spec.rb

```diff
@@ -1,135 +1,114 @@
-      context 'null id presented' do
-        let(:person_id) { nil }
-
-        it { expect { BlockScore::Person.retrieve(person_id) }.to raise_error(ArgumentError, 'ID must be supplied') }
-      end
-
-      context 'empty id presented' do
-        let(:person_id) { '' }
+      it { is_expected.to be_an_instance_of described_class }
+    end
+  end
 
-        it { expect { BlockScore::Person.retrieve(person_id) }.to raise_error(ArgumentError, 'ID must be supplied') }
-      end
+  describe '.retrieve' do
+    let(:person_id) { create(:person).id }
+    subject(:person) { described_class.retrieve(person_id) }
 
-      context 'Malformed ID presented' do
-        let(:person_id) { 'ABC&&234' }
+    it { is_expected.to be_persisted }
+    its(:name_first) { is_expected.not_to be_empty }
+    it { is_expected.to be_an_instance_of described_class }
+    context 'null id presented' do
+      let(:person_id) { nil }
 
-        it { expect { BlockScore::Person.retrieve(person_id) }.to raise_error(ArgumentError, 'ID is malformed') }
-      end
+      it { expect { described_class.retrieve(person_id) }.to raise_error(ArgumentError, 'ID must be supplied') }
     end
 
-    describe '.all' do
-      let(:uniq_token_one) { '4c2c019a1e3c33442644d9f52c7a93f7' }
-      let(:uniq_token_two) { 'b19a680671c6c006c6105aa450837668' }
-      subject(:people) { BlockScore::Person.all }
-
-      before do
-        create(:person, name_first: 'John', name_last: uniq_token_one)
-        create(:person, name_first: 'John', name_last: uniq_token_two)
-      end
+    context 'empty id presented' do
+      let(:person_id) { '' }
 
-      it { is_expected.not_to be_empty }
-      it { expect(people[0].name_last).to eq uniq_token_two }
-      it { expect(people[1].name_last).to eq uniq_token_one }
+      it { expect { described_class.retrieve(person_id) }.to raise_error(ArgumentError, 'ID must be supplied') }
     end
 
-    describe '#valid?' do
-      context 'valid person' do
-        subject(:person) { create(:valid_person) }
+    context 'Malformed ID presented' do
+      let(:person_id) { 'ABC&&234' }
 
-        it { is_expected.to be_persisted }
-        it { is_expected.to be_valid }
-        its(:class) { should be BlockScore::Person }
-      end
+      it { expect { described_class.retrieve(person_id) }.to raise_error(ArgumentError, 'ID is malformed') }
+    end
+  end
 
-      context 'invalid person' do
-        subject(:person) { create(:invalid_person) }
+  describe '.all' do
+    let(:uniq_token_one) { '4c2c019a1e3c33442644d9f52c7a93f7' }
+    let(:uniq_token_two) { 'b19a680671c6c006c6105aa450837668' }
+    subject(:people) { described_class.all }
 
-        it { is_expected.to be_persisted }
-        it { is_expected.not_to be_valid }
-        its(:class) { should be BlockScore::Person }
-      end
+    before do
+      create(:person, name_first: 'John', name_last: uniq_token_one)
+      create(:person, name_first: 'John', name_last: uniq_token_two)
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/68/files#r52838300